python/pylibcudf/pylibcudf/scalar.pyi

```diff
@@ -1,10 +1,19 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
+from typing import Any
+
 from pylibcudf.column import Column
 from pylibcudf.types import DataType
 
+NpDtype = type[Any]
```

Comment:
Nit: Could we rename this `NpGeneric`? 

Numpy does not have a stub for this type yet, but technically this should only accept `numpy.generic` https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.generic

Reply:
Yes done in https://github.com/rapidsai/cudf/pull/18415/commits/81084edbe9e4d4b32dce8c773b3ff80932eff650